scripts/jenkins/groovy/compareBenchmarksStage.groovy

```diff
@@ -360,8 +452,12 @@ def call(final pipelineContext, final stageConfig, final benchmarkFolderConfig)
                         interval = datasetValues[[Integer.parseInt(line.numRows), Integer.parseInt(line.numCols)]]
                         testCaseKey = 'dataset-size'
                         testCaseValue = "${line.numRows}x${line.numCols}"
+                    } else if (line.model_type) {
+                        interval = datasetValues[[Integer.parseInt(line.min_rule_length), Integer.parseInt(line.max_rule_length), line.model_type]]
```

Comment:
array needs to match the structure defined above - model_type is first

Reply:
fixed